src/HomeAssistantManager.cpp

```
@@ -937,6 +937,32 @@ void HomeAssistantManager::handleCommand(const char* topic, const char* payload)
         return;
     }
 
+    // tray_assign: loading blueprint assigns a scanned UID to a specific AMS tray
+    if (strcmp(command, "tray_assign") == 0) {
+        StaticJsonDocument<128> assignDoc;
+        if (deserializeJson(assignDoc, payload)) {
+            publishCommandResponse(command, false, "invalid_json");
+            return;
+        }
+
+        uint8_t trayIndex = assignDoc["tray_index"] | 0;
```

Clang (14.0.6)

[warning] variable 'trayIndex' is not initialized

(cppcoreguidelines-init-variables)